specs-derive/tests/saveload.rs

```diff
@@ -1,18 +1,21 @@
 #[macro_use]
 extern crate serde;
-extern crate specs;
+// Specs is renamed here so that the custom derive cannot refer specs directly
+extern crate specs as spocs;
 #[macro_use]
 extern crate specs_derive;
 
+use spocs::{Entity, saveload::{ConvertSaveload, Marker}, error::NoError};
```

Comment:
`spocs`

Reply:
Yes. Because I renamed `specs` to `spocs` at in extern crate. There is comment that explains why.

Reply:
Oh ah :smile:

Reply:
It's October,  obviously should be `spooks`.